specifications/api/swagger.yaml

```diff
@@ -102,15 +107,20 @@ walletMnemonicSentence: &walletMnemonicSentence
     format: bip-0039-mnemonic-word{english}
   example: ["squirrel", "material", "silly", "twice", "direct", "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"]
```

Comment:
small thing but this example has 12 words not 15 ;)

Reply:
Whoops.